storybook/vue/stories/component-list.ts

```diff
@@ -1,3 +1,13 @@
 const comps = import.meta.globEager('../../../components/*/vue/package.json')
 
+export const capName = Object.keys(comps).reduce((acc, fullPath) => {
```

Comment:
We should find a better name for this variable

`capitalizedComponentNameMap` ?